apps/web/src/components/pages/editor/de/editor-presentation-de.tsx

```diff
@@ -0,0 +1,301 @@
+import { editorRenderers } from '@editor/plugin/helpers/editor-renderer'
+import dynamic from 'next/dynamic'
+
+import { EditorContact } from '../editor-contact'
+import { EditorFeatures } from '../editor-features'
+import { EditorPartnerList, partners } from '../editor-partner-list'
+import { EditorRoadmap } from '../editor-roadmap'
+import { EditorTeam, teamDataVicky } from '../editor-team'
+import { Lazy } from '@/components/content/lazy'
+import { Link } from '@/components/content/link'
+import { HeadTags } from '@/components/head-tags'
+import { Logo } from '@/components/navigation/header/logo'
+import { breakpoints } from '@/helper/breakpoints'
+import { cn } from '@/helper/cn'
+import { createRenderers } from '@/serlo-editor-integration/create-renderers'
+
+const EducationPlugins = dynamic(() =>
+  import('../education-plugins').then((mod) => mod.EducationPlugins)
+)
+const h2Class =
+  'text-center text-4xl leading-cozy tracking-tight font-extrabold'
+const h3Class = 'text-gray-700 text-[1.3rem] font-extrabold'
+
+export function EditorPresentationDe() {
```

Comment:
for the futur: In a nice code we split strings / components and have something like

```typescript
interface EditorPresentationStrings {
  // strings
  title: string
  // sometimes we want to have a list of things
  benefits: string[]
}

export EditorPresentation({ strings: EditorPresentationstrings) {

}
```